src/components/premium-calculator/components/table/index.tsx

```diff
@@ -50,12 +49,24 @@ export const PremiumTable: React.FC<PremiumTableProps> = ({
 
           <tr className={styles.summary} style={{ gridTemplateColumns }}>
             <th>Premium</th>
-            {compound.map((value, index) => (
+            {compoundValues.map((value, index) => (
               <td key={index}>{formatMoneyString(value)}</td>
             ))}
           </tr>
         </tbody>
       </table>
+
+      <p>
```

Comment:
I think we need a little line-height adjustment here.

Reply:
Yes! rm'd the default bottom margin. Damn default html styles